### ISSUE TITLE
UI Overflow on email smtp setup page title - Mobile view

### ISSUE THREAD

**Issue by muzammildafedar** [muzammildafedar/udayah#13]
**Description:**
The above Instructions button is overlapping with the header text on the "Setup Your Email Using SMTP" page, causing a visual issue where the right side of the button extends beyond the intended area.

**Steps to Reproduce:**
Go to the Email Setup page.
Observe title overflow.

**Expected Behavior:**
The Instructions title should be properly aligned or flexible size and not overlap with any part of the header text.

**Actual Behavior:**
There is visible text showing a pixel overflow warning on the right side.

**Suggested Fix:**
Adjust the size of the title to ensure it is positioned correctly beneath the header text.

<img width="494" alt="Screenshot 2024-09-28 at 10 09 25 AM" src="https://github.com/user-attachments/assets/caacf94c-91a2-4360-a54c-474aa1954173">

**Comment by PartishtaP** [comment]
Heeyyy!!
Please assign me this.
I would like to work on it

**Comment by muzammildafedar** [comment]
> Heeyyy!! Please assign me this. I would like to work on it

Don't forget to clone the repository before starting. @PartishtaP Feel free to go ahead and take on the issue. Let me know if you need any help or clarification. Looking forward to your contributions!

**Comment by PartishtaP** [comment]
Heyy!!
can you please tell where is the code for the email set up page, like in which file.

**Comment by muzammildafedar** [comment]
> Heyy!! can you please tell where is the code for the email set up page, like in which file.

The SMTP page is located in `lib/widgets/smtp_page.dart`, where we use a custom widget called `CategoryBox` that includes a title prefix for all pages. In this widget, the `title` parameter is utilized between lines 53 and 56. Currently, `AppTextStyle` is defined, but you need to define another `AppTextStyle` specifically for mobile, which will reduce the font size. Additionally, you'll need to add a condition to adjust the font size based on whether the view is on mobile or desktop. See the example below:

```dart
Text(
  title,
  style: Responsive.isMobile(context) ? AppTextStyles.heading1BlackMobile : AppTextStyles.heading1Black,
),
```

Make sure to import:

```dart
import 'package:shippi/responsive.dart';
```

**Comment by thedevyash** [comment]
is this issue still open and if @PartishtaP not working on this anymore, can i work on this? 
@muzammildafedar

**Comment by muzammildafedar** [comment]
Yes, This issue still open. Feel free to go ahead and take on the issue.
> is this issue still open and if @PartishtaP not working on this anymore, can i work on this? @muzammildafedar

**Comment by may-tas** [comment]
I would like to work on this issue if its still open.

**Comment by Amitkumarkoli** [comment]
is this still open?? I want to work on this.

**Comment by muzammildafedar** [comment]
@thedevyash how's your progress on this? FYI, I'll need to un-assign you soon if I don't hear back

**Comment by tanayduddalwar** [comment]
@muzammildafedar please assign me this issue
